src/ert/job_queue/__init__.py

```diff
@@ -65,9 +65,9 @@ def setenv(var: str, value: str) -> None:
 
 
 def _load_lib() -> Any:
-    import ctypes
+    import ctypes  # noqa: PLC0415
```

Comment:
I think the `ctypes` import should be at the top of the file here if possible. `ert._clib` import can be done dynamically.